frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -151,24 +151,17 @@ export const RequestedChangesDisplay = ({
                 <h5 className="margin-bottom-1">Resources</h5>
                 <ul className={"margin-0"}>
                     <li>
-                        <NavLink
+                        <USLink
                             target="_blank"
```

Comment:
This one is notably _NOT_ an external link, but still opens `_blank`. Great example of the level of customization we have of the base component. The value add in something like `USExtLink` is the application of off-site link security measures and proper styling to indicate the link takes you away from ReportStream.

Reply:
Very small small point: I agree that we don't need `noreferrer` here since it's an internal link, but my feeling is that we should still add `rel="noopener"` whenever we have `target="_blank"` since that makes it so the new tab doesn't share the render process with the original tab.  It's not usually something we really have to worry about since the site isn't intensive on the client side, but I figure why not split off the processes anyway 🤷